packages/@react-spectrum/s2/src/ComboBox.tsx

```diff
@@ -148,6 +149,114 @@ const iconStyles = style({
   }
 });
 
+export let listbox = style<{size: 'S' | 'M' | 'L' | 'XL'}>({
```

Comment:
What are the differences between the styles here and the ones in menu? A bit unfortunate to duplicate them

Reply:
Mainly that menu has some additional styles that Combobox and Picker didn't need but it wasn't actually affecting anything. I could probably revert the change and just use menu but I'd need to double check